featuregates/client/pkg/util/featuregate.go

```diff
@@ -82,7 +86,68 @@ func ComputeFeatureStates(featureGateSpec configv1alpha1.FeatureGateSpec, featur
 	return activated, deactivated, unavailable
 }
 
+// ComputeFeatureStatesForCoreAPIGroupResources takes a corev1alpha2 FeatureGate spec and computes the actual state (activated, deactivated or unavailable)
```

Comment:
Nit: what is our repo's style guide for line width? Whatever it is, these lines here (89-90) and the ones above (30-33) might exceed it.